ffi/src/winscard/mod.rs

```diff
@@ -1,3 +1,5 @@
+#![cfg(all(target_os = "windows", feature = "scard"))]
```

Comment:
currently, we should be able to compile this module on non-Windows OS. Do you have any problems with that?